<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Pelanggan - Aplikasi Jasa Makeup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameSidebar">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#frameSidebar {
    background-color: #E91E63;
}

QPushButton {
    background-color: transparent;
    color: white;
    text-align: left;
    padding-left: 20px;
    border: none;
    font-size: 11pt;
}

QPushButton:hover {
    background-color: #C2185B;
}

QLabel {
    color: white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblLogo">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>üíÑ MAKEUP APP</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDashboard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üè† Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPelanggan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üë• Data Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLayanan">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>‚ú® Layanan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnJadwal">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üìÖ Jadwal Booking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTransaksi">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üõí Transaksi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPembayaran">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>üí≥ Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetContent" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frameHeader">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#frameHeader {
    background-color: white;
    border-bottom: 1px solid #ddd;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblPageTitle">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Data Pelanggan</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblUsername">
            <property name="text">
             <string>üë§ Admin User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogout">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
}
QPushButton:hover {
    background-color: #da190b;
}</string>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="pageContent" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#pageContent {
    background-color: #f5f5f5;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frameSearch">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="txtSearchPelanggan">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #ddd;
    border-radius: 5px;
    padding: 8px;
    background-color: white;
}
QLineEdit:focus {
    border: 2px solid #E91E63;
}</string>
               </property>
               <property name="placeholderText">
                <string>üîç Cari pelanggan...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnTambahPelanggan">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #E91E63;
    color: white;
    padding: 10px 20px;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #C2185B;
}</string>
               </property>
               <property name="text">
                <string>‚ûï Tambah Pelanggan</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tablePelanggan">
            <property name="styleSheet">
             <string notr="true">QTableWidget {
    background-color: white;
    border: 1px solid #ddd;
    gridline-color: #f0f0f0;
}
QTableWidget::item:selected {
    background-color: #FCE4EC;
    color: #333;
}
QHeaderView::section {
    background-color: #E91E63;
    color: white;
    padding: 8px;
    border: none;
    font-weight: bold;
}</string>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nama Lengkap</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>No HP</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Alamat</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Aksi</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupFormPelanggan">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 2px solid #E91E63;
    border-radius: 8px;
    margin-top: 10px;
    padding: 15px;
    background-color: white;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    background-color: white;
    color: #E91E63;
}</string>
            </property>
            <property name="title">
             <string>üìù Form Data Pelanggan</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Nama Lengkap:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="txtNamaPelanggan">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
               </property>
               <property name="placeholderText">
                <string>Masukkan nama lengkap</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>No HP:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="txtNoHPPelanggan">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
               </property>
               <property name="placeholderText">
                <string>08xxxxxxxxxx</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Alamat:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QTextEdit" name="txtAlamatPelanggan">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>80</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QTextEdit {
    border: 1px solid #ddd;
    border-radius: 4px;
    padding: 8px;
}</string>
               </property>
               <property name="placeholderText">
                <string>Masukkan alamat lengkap</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QWidget" name="widget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="btnSimpanPelanggan">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;
}</string>
                  </property>
                  <property name="text">
                   <string>üíæ Simpan</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnBatalPelanggan">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #da190b;
}</string>
                  </property>
                  <property name="text">
                   <string>‚ùå Batal</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
